packages/libz/libz.spec

```
@@ -0,0 +1,60 @@
+Name: %{_cross_os}libz
+Version: 1.2.11
+Release: 1%{?dist}
+Summary: Compression and decompression library
+URL: https://www.zlib.net/
+License: zlib and Boost
+
+Source: http://www.zlib.net/zlib-%{version}.tar.gz
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: Header files and libraries for Zlib development
+Requires: %{name}
+
+%description devel
+%{summary}.
+
+%prep
+%setup -n zlib-%{version}
+
+# Sets cross build flags, target cross compiler, and env variables
+# required to `make install` libz
+%global set_env \
+%set_cross_build_flags \\\
+export CC="%{_cross_target}-gcc" \\\
+export DESTDIR='%{buildroot}' \\\
+export PREFIX='%{_cross_prefix}' \\\
+export SHLIBDIR='%{_cross_libdir}' \\\
+%{nil}
+
+%build
+%set_env
+# Force --prefix=$PREFIX
+./configure --prefix=$PREFIX
+%make_build
+
+%check
+make test
+
+%install
+%set_env
+%make_install
+# We don't want the static libraries
+rm %{buildroot}/%{_cross_libdir}/libz.a
+
+%files
+%license README
+%{_cross_attribution_file}
+%{_cross_libdir}/libz.so.*
+%exclude %{_cross_mandir}
+
+%files devel
+%{_cross_libdir}/libz.so
+%{_cross_libdir}/pkgconfig/zlib.pc
+%{_cross_includedir}/zlib.h
+%{_cross_includedir}/zconf.h
```

nit: I tend to use boilerplate for these includes, rather than package-specific ones:
```suggestion
%files
%license README
%{_cross_attribution_file}
%{_cross_libdir}/*.so.*
%exclude %{_cross_mandir}

%files devel
%{_cross_libdir}/*.so
%{_cross_includedir}/*.h
%{_cross_pkgconfigdir}/*.pc
```